instrumentation/logback/logback-appender-1.0/javaagent/src/test/groovy/LogbackTest.groovy

```diff
@@ -44,6 +44,13 @@ class LogbackTest extends AgentInstrumentationSpecification {
       waitForTraces(1)
     }
 
+    String jvmVersion = System.getProperty("java.vm.specification.version")
+    int codeAttributes = 3
+    boolean jvmVersionGreaterThanOrEqualTo18 = jvmVersion.startsWith("1.8") && Integer.parseInt(jvmVersion) >= 18
```

Comment:
It's the jvm version (not the code version) Some logback tests are failing because logback has a different behavior regarding the line number for a JVM 18